rerun_js/web-viewer/index.js

```diff
@@ -88,14 +88,16 @@ export class WebViewer {
    * @see {WebViewer.start}
    *
    * @param {string | string[]} rrd URLs to `.rrd` files or WebSocket connections to our SDK.
+   * @param {boolean} [follow_if_http] Whether Rerun should open the resource in "Following" mode when streaming
+   *        from an HTTP url. Defaults to `false`. Ignored for non-HTTP URLs.
    */
-  open(rrd) {
+  open(rrd, follow_if_http = false) {
```

Comment:
Putting a flag this specific onto a generic-sounding `open` call makes me uneasy, but it's fine if we plan to remove it at some point

Reply:
Hmm, yeah. These things tend to last longer than we'd like though.

Anything we could do to improve?

Would it be better as a different API like `open_and_follow`?

Reply:
I'm not sure 😅. Maybe this could be in an options object:
```js
  /** @param {{ follow_if_http?: boolean }} options */
  open(rrd, options) {
    // ...
  }
```
This way we can add more flags in a backwards compatible way